.travis.yml

```diff
@@ -4,7 +4,7 @@ git:
 language: node_js
 node_js:
   - node
-  - '4'
+  - 'lts/*'
```

Comment:
Don't include unrelated changes.

Reply:
It's not unrelated. node.js 4 is really old and doesn't work with current versions of dependencies